utils/bels.json

```diff
@@ -0,0 +1,172 @@
+{
```

Comment:
Why is this in utils?  It belongs to xc7?

Reply:
yes it should be in xc7. I'll move it there

Reply:
done. The file is now in xc7/bels.json